optimum/intel/openvino/configuration.py

```diff
@@ -775,3 +775,348 @@ def to_dict(self) -> Dict[str, Any]:
 
     def to_diff_dict(self) -> Dict[str, Any]:
         return self._to_dict_safe(to_diff_dict=True)
+
+
+class OVCompressWeightsOptions:
```

Comment:
```suggestion
class OVWeightsCompressionOptions:
```

Reply:
Let's use this `OVXXXOptions` classes as base entities for corresponding `OVXXXConfig`s.